ui/apps/everest/src/components/confirm-dialog-namespace/confirm-dialog-namespace.tsx

```diff
@@ -0,0 +1,49 @@
+import { Button, Dialog, DialogActions, DialogContent } from '@mui/material';
+import { DialogTitle } from '@percona/ui-lib';
+import { kebabize } from '@percona/utils';
+import { ConfirmDialogNamespaceProps } from './confirm-dialog-namespace.types';
+
+export const ConfirmDialogNamespace = ({
```

Comment:
Why we can't use standart confirm dialog with customization via props?

Reply:
I duplicated this component because the way we handle confirm for these two resources has to include a namespace param (which we don't need for the others; the hooks need to be sent the namespace)
```
  const onClick = () => {
    handleConfirm(selectedId, selectedNamespace);
  };
```
But now that you mentioned prop customization, I think we can have two separate props `selectedNamespace` and `handleConfirmNamespace` and use those in these two cases, instead of duplicating the component.. What do you say about this approach?